action.yml

```diff
@@ -12,11 +12,23 @@ inputs:
   service-account-password:
     description: 'The service account password'
     required: true
+  web-site-name:
+    description: 'The name of the web site'
+    default: ''
+    required: false
+  web-site-host-header:
+    description: 'The host-header the web site should respond to'
+    default: ''
+    required: false
+  web-site-path:
+    description: 'The local directory of the web site, i.e., "c:\inetpub\webapp"'
+    default: ''
+    required: false
```

Comment:
These should probably be added to the Inputs section of the README. In addition, do you think it would make sense to note that these are required if you specify `create-site` for the `action` input?